docs/pages/concepts/source.md

```diff
@@ -26,6 +26,11 @@ table named `user` within the Postgres database. And that's it - once this
 is written, `UserLocation` dataset will start mirroring your postgres table
 `user`and will update as the underlying Postgres table updates.
 
+### Bounded
+The `bounded` attribute in the source indicates whether the source possesses a finite amount of data that does not expand. It is assigned a value of True when the source has limited data, with the default value being False.
```

Comment:
I think we should also share the idea behind this. Why does the user care if the source is bounded or not? 

The reason is for watermarks. I think we should write about it, but not too much in detail. We should say something in the lines of -
this ensures correct watermark values are propagated to the downstream pipelines. Without this, say an unbounded source with no data flow will cause the watermarks to be stuck and will have cost and performance implications.